README.md

```diff
@@ -17,6 +17,7 @@ It features __randomizable simulations__ written __in standalone Python__ (no li
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-green.svg)](https://opensource.org/licenses/BSD-3-Clause)
 [![Maintainability](https://api.codeclimate.com/v1/badges/73ad23bdf05361b022b4/maintainability)](https://codeclimate.com/github/famura/SimuRLacra/maintainability)
 [![Documentation](https://github.com/famura/SimuRLacra/workflows/Documentation/badge.svg?branch=master)](https://famura.github.io/SimuRLacra/)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-32CD32.svg)](https://github.com/psf/black)
```

Comment:
what? lol pls delete that line ince it is basically the same as the one two lines below